engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/settings/PlayerSettingsScreen.java

```diff
@@ -88,6 +84,7 @@
     private UISlider heightSlider;
     private UISlider eyeHeightSlider;
     private UIImage img;
+    private UICheckbox discordpresence;
```

Comment:
`discordpresence` -> `discordPresence`

Reply:
**Done** I didn't notice the name. Sorry, 😅 Thanks for telling me about it.